peps/pep-9999.rst

```diff
@@ -0,0 +1,616 @@
+PEP: 9999
+Title: Standardizing selection of Python package files across multiple sources
+Author: Michael Sarahan, msarahan@gmail.com
+Sponsor: Barry Warsaw, barry@python.org
+PEP-Delegate: <PEP delegate’s real name>
+Discussions-To: <REQUIRED: URL of current canonical discussion thread>
+Status: Draft
+Type: Standards Track
```

Comment:
It seems like this could either be a [Standards Track or Information PEP](https://peps.python.org/pep-0001/#pep-types).  While it's proposing a standard, that standard isn't directly for Python, and tool implementers can follow the advice or not.  I'd lean more towards Informational, but if you feel strongly about ST, then that's fine with me.

Reply:
I went with Information. Thanks.